src/modules/fare-contracts/details/Barcode.tsx

```diff
@@ -80,9 +92,53 @@ function useScreenBrightnessIncrease() {
   );
 }
 
+const BarcodeInspectionView = () => {
+  const styles = useStyles();
+  const {
+    aztec_code_size_in_cm,
+    aztec_code_padding,
+    enable_new_token_barcode_base64,
+  } = useRemoteConfigContext();
+
+  useEffect(() => {
+    (async () => {
+      try {
+        // Prepare data for RNBarcodeInspectionView
+        await beginInspection(CONTEXT_ID, {
+          visualInspectionNonces: undefined,
+          includeCertificate: false,
+          deviceDetails: {
+            application: false,
+            os: false,
+            network: false,
+            bluetooth: true,
+          },
```

Comment:
I guess it is easiest to just check what Entur does 🤷 Hard to speculate in the effect of each setting. But really nice to see the different options I think